modules/clightning-db-replication.nix

```diff
@@ -0,0 +1,221 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  options.services.clightning.replication = {
+    enable =  mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable live replication of the clightning database.
+        This ensures that funds are safe at all times by immediately backing up
+        each channel update.
```

Comment:
Ensuring that funds are safe is hyperbole if only for a reason that you can't roll back. Perhaps "This prevents losing off-chain funds when the primary wallet file becomes inaccessible."

Reply:
Fixed in 8dd8bd252f08948bd163d5b9bd250189781a00a8